src/components/editMeasure/details/MeasureDefinitions/MeasureDefinitions.tsx

```diff
@@ -145,6 +266,42 @@ const MeasureDefinitions = (props: MeasureDefinitionsProps) => {
   };
   const handleClearClick = () => {};
 
+  const deleteMeasureDefinition = (id) => {
```

Comment:
I think this could be wrapped in useCallback since this is passed to child component

Reply:
@adongare done.